packages/components/pie-select/src/index.ts

```diff
@@ -83,6 +86,29 @@ export class PieSelect extends FormControlMixin(RtlMixin(PieElement)) implements
         this._internals.setFormValue(this._select.value);
     }
 
+    @property()
+    public get value (): SelectProps['value'] {
+        // If no value was assigned
+        // and the select element is available
+        // return its value as by default it will pick the first available option
+        if (this._value === '') {
+            if (!this._select) {
+                return '';
+            }
+            return this._select.value;
+        }
+
+        return this._value;
+    }
+
+    public set value (newValue: SelectProps['value']) {
+        const safeNewValue = newValue ? String(newValue) : '';
+        this._internals.setFormValue(safeNewValue);
+        this._value = safeNewValue;
+
+        this.requestUpdate();
```

Comment:
do we still need to call requestUpdate if we already use the @property() decorator? 
https://lit.dev/docs/components/properties/#accessors-custom

Reply:
Sorry just saw you made a follow-up with #2552